lua/calvera/theme.lua

```diff
@@ -108,7 +108,7 @@ theme.loadEditor = function ()
 		FoldColumn =			{ fg = calvera.blue }, -- 'foldcolumn'
 		IncSearch =				{ fg = calvera.highlight, bg = calvera.white, style = 'reverse' }, -- 'incsearch' highlighting; also used for the text replaced with ":s///c"
 		LineNr =				{ fg = calvera.line_numbers }, -- Line number for ":number" and ":#" commands, and when 'number' or 'relativenumber' option is set.
-		CursorLineNr =			{ fg = calvera.accent }, -- Like LineNr when 'cursorline' or 'relativenumber' is set for the cursor line.
+		CursorLineNr =			{ fg = calvera.white }, -- Like LineNr when 'cursorline' or 'relativenumber' is set for the cursor line.
```

Comment:
calvera.blue would look much nicer in my opinion

Reply:
This was an unintended change tbh, I will definitely try blue and compare it with accent.

Reply:
🤔 I think changing `LineNr` to accent will be better too!